wakuv2/waku.go

```diff
@@ -1176,6 +1182,7 @@ func (w *Waku) Start() error {
 					if err != nil {
 						w.logger.Error("OnNewEnvelopes error", zap.Error(err))
 					}
+					w.logger.Info("Got a missing message!")
```

Comment:
There's a convention in Go to begin log messages with lower case letter.
```suggestion
					w.logger.Info("got a missing message!")
```